Flick/Models/Notification.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  Notification.swift
+//  Flick
+//
+//  Created by Lucy Xu on 8/8/20.
+//  Copyright © 2020 flick. All rights reserved.
+//
+
+enum Notification {
+    case FriendRequest(fromUser: String, type: FriendRequest.FriendRequestType)
+    case CollaborationInvite(fromUser: String, media: String)
+    case ListActivity(fromUser: String, list: String)
+    case ActivityLike(fromUser: String, likedContent: ActivityLike.ActivityLikeType, media: String)
```

Comment:
i think the naming convention for enum cases is first letter lowercased? it looks kinda weird to me these cases start capitalized lol

Reply:
oh this is because its like a constructor so think it has to be this way